apps/server/src/modules/user-login-migration/controller/api-test/user-login-migration.api.spec.ts

```diff
@@ -189,19 +189,21 @@ describe('UserLoginMigrationController (API)', () => {
 			});
 		});
 
-		describe('when current User start the migration and is not authorized', () => {
-			const setup = () => {
-				currentUser = undefined;
+		describe('when invalid User start the migration', () => {
+			const setup = async () => {
+				const invalidUser = userFactory.build();
+				await em.persistAndFlush([invalidUser]);
+				currentUser = mapUserToCurrentUser(invalidUser);
```

Comment:
We have implement a solution that avoid the currentUser hack. Please use TestApiClient instead. You find already implemented solutions in code.